lib/Dialect/Torch/Transforms/ReduceOpVariants.cpp

```diff
@@ -126,11 +126,16 @@ class ReduceNonValueSemanticOps : public RewritePattern {
       : RewritePattern(MatchAnyOpTypeTag(), /*benefit=*/1, context) {}
   LogicalResult matchAndRewrite(Operation *op,
                                 PatternRewriter &rewriter) const override {
-    if (!isa<AtenUniform_Op>(op))
+    Operation *newOp;
+    if (isa<AtenUniform_Op>(op))
+      newOp = rewriter.create<PseudoAtenUniformOp>(
+          op->getLoc(), op->getResultTypes(), op->getOperands());
+    else if (isa<AtenBernoulli_FloatOp>(op))
+      newOp = rewriter.create<PseudoAtenBernoulliFloatOp>(
+          op->getLoc(), op->getResultTypes(), op->getOperands());
```

Comment:
nit: this was in the existing code as well. Can you help to assign `op->getLoc()` to a `loc` variable.